var/spack/repos/builtin/packages/apex/package.py

```diff
@@ -4,30 +4,78 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack import *
+import sys
 
 
 class Apex(CMakePackage):
     """Autonomic Performance Environment for eXascale (APEX)."""
 
-    homepage = "http://github.com/khuck/xpress-apex"
-    url      = "http://github.com/khuck/xpress-apex/archive/v0.1.tar.gz"
+    maintainers = ['khuck']
+    homepage = "https://github.com/khuck/xpress-apex"
+    url      = "https://github.com/khuck/xpress-apex/archive/v2.2.0.tar.gz"
+    git      = "https://github.com/khuck/xpress-apex"
 
-    version('0.1',   sha256='efd10f38a61ebdb9f8adee9dc84acaee22d065b2e6eea1888872a7bfca0f4495')
+    version('develop', branch='develop')
+    version('master', branch='master')
+    version('2.2.0', sha256='cd5eddb1f6d26b7dbb4a8afeca2aa28036c7d0987e0af0400f4f96733889c75c')
 
-    depends_on("binutils+libiberty@:2.33.1")
-    depends_on("boost@1.54:")
-    depends_on('cmake@2.8.12:', type='build')
-    depends_on("activeharmony@4.5:")
-    depends_on("ompt-openmp")
+    # Disable some default dependencies on Darwin/OSX
+    darwin_default = False
+    if sys.platform != 'darwin':
+        darwin_default = True
+
+    # Enable by default
+    variant('activeharmony', default=True, description='Enables Active Harmony support')
+    variant('binutils', default=True, description='Enables Binutils support')
```

Comment:
What kind of binutils support is this? All packages use binutils, but we try not to add a dependency on binutils to every package.

Reply:
APEX uses bfd to resolve function pointer addresses, and demangle.h for demangling.  Actually, this should be similar to the TAU dependency, so I just now realize I need to add some variants to that dependency...

Reply:
oops, just realized I do have the depends_on correct already, disregard the comment about additional changes to that dependency...